### PR TITLE
Makes termites accully attack the drill

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
Vastly lowers the danger of mining drills termites
Makes mining drill termites properly walk up and attack the drill

### CODE REVIEW

File: code/modules/mining/drilling/drill.dm [hunk 1]
```diff
@@ -51,10 +51,14 @@
 /obj/machinery/mining/drill/Destroy()
 	for(var/obj/machinery/mining/brace/b in supports)
 		b.disconnect()
+	qdel(soul)
+	soul = null
```
Comment (nikothedude) [inline code/modules/mining/drilling/drill.dm:55]: You can replace this with QDEL_NULL(soul), same effect

Comment (Trilbyspaceclone) [reply code/modules/mining/drilling/drill.dm:55]: thats spooky way of doing it

Comment (nikothedude) [reply code/modules/mining/drilling/drill.dm:55]: its a cleaner way

**Comment by cdb-is-not-good** [comment]
Tested 'n working?

**Pull request merged**